src/parser.rs

```diff
@@ -906,6 +926,40 @@ impl Parser {
         })
     }
 
+    /// Parse Create User Defined Function statement
+    ///
+    /// 1. https://jakewheat.github.io/sql-overview/sql-2011-foundation-grammar.html#schema-function
+    /// 2. https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions
+    pub fn parse_create_function(&mut self, or_replace: bool) -> Result<Statement, ParserError> {
+        let if_not_exists = self.parse_keywords(vec!["IF", "NOT", "EXISTS"]);
+
+        let name = self.parse_object_name()?;
+        self.expect_token(&Token::LParen)?;
+        let args = self.parse_optional_parameters()?; // TODO: support arguments
+
+        let temporary = self.parse_keyword("TEMPORARY") || self.parse_keyword("TEMP");
```

Comment:
Why is this here? The [doc you linked to](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions) has this: `CREATE [OR REPLACE] {TEMPORARY | TEMP} FUNCTION`. Would you add a test for this as well?

Reply:
Sure will do.